WWW/minidata_batch.json

```diff
@@ -1,4 +1,4 @@
 {
-  
-  "minidata_test_alignment.sampleFastq": "/fh/scratch/delete90/_DaSL/WDL_mini_data/NCIH2172_LUNG/SRR8618962_combined.fastq"
+  "minidata_test_alignment.base_file_name": "NA_12878_20k",
```

Comment:
R/E: https://github.com/fhdsl/WDL-sandbox/pull/1#issuecomment-1868021508 either option

Essentially, this is what I recommend avoiding. The user having to input one string for one sample isn't unreasonable, but if you were processing lots of samples, this would quickly become tedious.